### ISSUE TITLE
Shipped XRootD CMake configuration doesn't work

### ISSUE THREAD

**Issue by bbockelm** [xrootd/xrootd#1194]
Tried using the shipped CMake config today for a new module I was working on -- effectively, it doesn't appear to work.

First, when one does:

```
find_package( xrootd REQUIRED )
```

On RHEL it'll search in `/usr/share/xrootd/cmake/` for files named `xrootdConfig.cmake` or `xrootd-config.cmake` (case-sensitive).  The file is currently named `FindXRootD.cmake`, which will never match.  If you want:

```
find_package( XRootD REQUIRED )
```

then we need to move the file to `/usr/share/XRootD/cmake/XRootD-config.cmake`.

We also need to have the `_FOUND` variables to always be set and match case (i.e., `XRootD_FOUND` should always be set to true or false).

Finally, there's a typo breaking the `XROOTD_PLUGIN_VERSION` mechanism -- there's an extra comma `,` after the variable name.  This appears to work:
```
SET( XROOTD_PLUGIN_VERSION 4 )
```

**Comment by simonmichal** [comment]
Hi Brian,

I replaced the `FindXRootD.cmake` with `XRootDConfig.cmake` in 4.12.0:
https://github.com/xrootd/xrootd/commit/25bf0ba2fbcbc9007db930f717002c0d5ad79daf
https://github.com/xrootd/xrootd/blob/master/docs/ReleaseNotes.txt#L49

Against which version were you testing?

Regarding the `*_FOUND` variables and the disruptive comma, this shall be fixed!

Cheers,
Michal

**Comment by simonmichal** [comment]
@bbockelm : I just pushed 4fba4a188baffa411a4c145355381982dc613e83, it should fix the problems you pointed out.

Could you give it a try? If you build the project you will find the XRootD config module at `build/cmake/XRootDConfig.cmake`, by default it is installed in `/usr/share/xrootd/cmake/XRootDConfig.cmake`.

Thanks!

**Comment by bbockelm** [comment]
@simonmichal - https://github.com/xrootd/xrootd/commit/4fba4a188baffa411a4c145355381982dc613e83 has a misspelling in the first hunk.  `SER` should be `SET`.

**Comment by simonmichal** [comment]
@bbockelm : fixed: a585e5fd598cfd983109a5aa10a36f6090363fd3, thanks for pointing it out!

**Comment by abh3** [comment]
I am closing this as it appears to be fixed.
